### ISSUE TITLE
Notify users about pinning Firefox in Windows

### ISSUE THREAD

**Issue by slightlyoffbeat** [mozilla/bedrock#11759]
## Description

There is a request to notify users about pinning Firefox for Windows 7, 10, and 11. 

We are unable to notify users in firstrun, so the request is to include a SUMO link on the /thanks page.

Details:
- Go-Live by fx103
- Show to Windows 7, 10, and 11 users
- Keep as unobtrusive as possible

Legal approved copy: `Firefox will be pinned to the Windows taskbar after installation`

Link: `https://support.mozilla.org/en-US/kb/how-unpin-firefox-taskbar-windows-10`

Mock:

<img width="1312" alt="Screen Shot 2022-06-21 at 11 17 45 AM" src="https://user-images.githubusercontent.com/2030656/174870784-73a7b6d5-cce3-4975-adeb-979270f0cdc5.png">

Requirements:

- [ ] only show to windows 11 & windows 10
- [ ] only show to firefox 103 and above
- [ ] Remove windows 10 logo or replace windows 10 logo with windows 11 logo when user is on windows 11 (currently shows windows 10 logo to win 11 users).

**Comment by slightlyoffbeat** [comment]
Proposed SUMO link is https://support.mozilla.org/en-US/kb/how-unpin-firefox-taskbar-windows-10 

We are working with SUMO to update the article to include Windows 7 and 11.

**Comment by slightlyoffbeat** [comment]
@alexgibson we should hold on merging. Still confirming details with legal and product team. I updated this issue with the mock, copy, and link that will most likely be final if contributor wants to update. Also added requirements.

**Comment by slightlyoffbeat** [comment]
Added a requirement around the windows 10/11 logo, but we can split into another issue if that's better.

**Comment by alexgibson** [comment]
Thanks @slightlyoffbeat. I think it probably makes sense for someone on our team to take this on and finish off the finer details. 

I’ll pick this one up but, hold off pending final approvals.

**Comment by rtestard** [comment]
A couple comments:
- Can we remove "only show to firefox 103 and above" since this should be shown to all downloading clients starting on release 103 date
- Could we have a Win11 version of https://support.mozilla.org/fr/kb/windows-10-avertit-utiliser-application-verifiee-microsoft ?

**Comment by alexgibson** [comment]
> Can we remove "only show to firefox 103 and above" since this should be shown to all downloading clients starting on release 103 date

I think it makes sense to use a feature switch for this, so we can display the message to everyone after the 103 release date.

> Could we have a Win11 version of https://support.mozilla.org/fr/kb/windows-10-avertit-utiliser-application-verifiee-microsoft ?

Accurately detecting Windows 11 may be problematic, since all modern browsers have now frozen their UA strings as "Windows 10". We could use Client Hints, but that's not currently supported in Firefox.

Perhaps we can simplify this into a single SUMO page?

**Comment by alexgibson** [comment]
Filed https://github.com/mozilla/bedrock/issues/11799 to update the logo mentioned above

**Comment by alexgibson** [comment]
@rtestard @slightlyoffbeat I notice that https://support.mozilla.org/en-US/kb/how-unpin-firefox-taskbar-windows-10 has instructions both both Windows 10 and 11. Maybe we can just update the title / URL to make it more obvious?

**Comment by alexgibson** [comment]
@slightlyoffbeat I have a draft PR ready for this. The link/copy is behind a switch, so that we can toggle it on when Firefox 103 releases.

I suggest that if we can finalise the link text and URL for now, then we can review/merge this and at least get the new copy translated well ahead of time, whilst the SUMO page is finalised.

**Comment by amirmozla** [comment]
@slightlyoffbeat , Should we move forward with the link and text attached in the description to unblock this ticket?

**Comment by alexgibson** [comment]
@rtestard I think we're just waiting on confirmation that the SUMO link is final / ready. I have a draft PR ready, so as soon as we get confirmation we can get this reviewed and pushed to prod ready.

**Comment by alexgibson** [comment]
I've marked the PR as ready for code review here, but it would be great to get @slightlyoffbeat to chime in and confirm we're ok to move forward with:

- Link copy: Firefox will be pinned to the Windows taskbar after installation
- Link URL: https://support.mozilla.org/en-US/kb/how-unpin-firefox-taskbar-windows-10

This will be shown by default to users on Windows, so will cover everyone including Windows 7, 8, 10, 11.

Note: we can no longer reliably detect windows versions using some browsers, so this seems like the easiest route.
